README.md

```diff
@@ -126,6 +126,11 @@ Composer shipped as jar, to run it you need JVM 1.8+: `java -jar composer-latest
   * Requires test orchestrator & test services APKs to be installed on device before executing.
   * More info: https://developer.android.com/training/testing/junit-runner#using-android-test-orchestrator
   * Example: `--with-orchestrator true`
+* `--extra-apks`
+  * Apks to be installed for utilities. What you would typically declare in gradle as `androidTestUtil` 
+  * Default: empty, only apk and test apk would be installed.
+  * Works great with orchestrator to install orchestrator & test services APKs.
```

Comment:
nit: Orchestrator with capital O, can you please also make it a link to Android docs about it?

Reply:
The link is just two lines before this, on the --with-orchestrator flag.